historical/README

```diff
@@ -3558,7 +3558,7 @@ WLALINK (the linker).
 
 WLA has four separate phases, called a little bit incorrectly passes:
```

Comment:
After this PR they are called phases so they are not incorrectly called passes any more :)

Reply:
Just this might be enough:

"WLA has four separate phases:"

:)